### ISSUE TITLE
Make GraphiQL.Menu keyboard accessible

### ISSUE THREAD

**Issue by wincent** [graphql/graphiql#327]
For a11y.

**Comment by AGS-** [comment]
#336

**Comment by asiandrummer** [comment]
Maybe I'm missing/forgetting something, but why was this needed?

**Comment by AGS-** [comment]
Oh, well since we went ahead and made the menus and selects stay open on click, @wincent and I were thinking that this would be something nice to have that takes up a trivial amount of time to do.

**Comment by wincent** [comment]
> Maybe I'm missing/forgetting something, but why was this needed?

Keeping things [accessible](https://en.wikipedia.org/wiki/Web_accessibility) is a best practice. It's pretty common to fail to do so when replacing native browser elements with hand-rolled custom versions.

**Comment by asiandrummer** [comment]
Ahhhhh, I misunderstood the word `accessible`. Makes sense!

**Comment by acao** [comment]
@wincent we recently had an issue and a first time contrib PR that should have addressed this and other accessibility issues, should have marked this as a duplicate. want to confirm?

**Comment by benjie** [comment]
I'm going to close this as "we believe it's fixed" but if someone wants to chime in that it isn't we should definitely re-open.

Specifically ToolbarButton was made accessible in this PR: https://github.com/graphql/graphiql/pull/967/files#diff-07f119989c064b0e2a2b0883e992317d
